packages/opentelemetry-tracing/src/types.ts

```diff
@@ -44,6 +50,19 @@ export interface TracerConfig {
   traceParams?: TraceParams;
 }
 
+/**
+ * Configuration options for registering the API with the SDK.
+ * Undefined values may be substituted for defaults, and null
+ * values will not be registered.
+ */
+export interface SDKRegistrationConfig {
+  /** Propagator to register as the global propagator */
+  propagator?: HttpTextFormat | null;
```

Comment:
should this be `undefined` instead of `null`

Reply:
undefined and null have different semantics. If you pass undefined (or omit a property) it will use a default. If you pass null, it will be skipped.

Reply:
If this answer satisfies you please mark it as resolved.